layouts/faq/section.html

```diff
@@ -0,0 +1,26 @@
+{{ define "head-suffix" }}
+{{ $data := index .Site.Data .Site.Language.Lang }}
+  {{ with $data.faq }}
+    <script type="application/ld+json">
+    {{ . }}
+    </script>
+  {{ end }}
+{{ end }}
+{{ define "breadcrumbs"}}{{ partial "breadcrumbs.html" . }}{{end}}
+{{ define "main" }}
```

Comment:
I would prefer not to override main here.

Can we move this functionality to a shortcode that we can use in markdown file?